instrumentation/kafka-clients/kafka-clients-0.11/bootstrap/src/main/java/io/opentelemetry/javaagent/bootstrap/kafka/KafkaClientsConsumerProcessTracing.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.bootstrap.kafka;
+
+public final class KafkaClientsConsumerProcessTracing {
```

Comment:
can you add a brief comment about why this lives in the bootstrap class loader?

Reply:
added a comment, could you check whether it makes sense

Reply:
thx 👍